pkgs/games/anki/addons/anki-button-colours/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "anki-button-colours";
+  version = "2.1.1";
```

Comment:
There's a lot missing from the latest tag. You might want to do the latest commit.